### PR TITLE
Change clearfix extend to a mixin, update refs

### PR DESCRIPTION
**Pull request by josephco**
This will be a minor update to 1.5.x
This PR must be merged before the corresponding PR against Casper repo (https://github.com/CasperSleep/Casper/pull/2677)

### CODE REVIEW

File: utilities/_clearfix.scss [hunk 1]
```diff
@@ -4,17 +4,18 @@
...
-    &:after {
-        content: "";
-        display: table;
-        clear: both;
-    }
-
-}
+ .clearfix {
+   @include clearfix;
+ }
```
Comment (robwierzbowski) [inline utilities/_clearfix.scss]: As long as we're making this change, can we remove the class entirely?

**Comment by robwierzbowski** [comment]
One question, everything else LGTM.

File: utilities/_clearfix.scss (hunk 1 above)
Comment (robwierzbowski) [inline utilities/_clearfix.scss]: Talked IRL, tabling discussion for now.

**Comment by josephco** [comment]
@robwierzbowski made updates to move the mixin into display_helpers, and updated references in Casper repo here: https://github.com/CasperSleep/Casper/pull/2677/files

**Comment by robwierzbowski** [comment]
Nice. Still DTM.

**Pull request merged**